Docs.rs fails to build on 0.20.0

Due to recent removal of `doc_auto_cfg`, docs.rs build on `0.20.0` fails with:

```txt
[INFO] [stderr] error[E0557]: feature has been removed
[INFO] [stderr]   --> src/lib.rs:19:29
[INFO] [stderr]    |
[INFO] [stderr] 19 | #![cfg_attr(docsrs, feature(doc_auto_cfg))]
[INFO] [stderr]    |                             ^^^^^^^^^^^^ feature has been removed
[INFO] [stderr]    |
[INFO] [stderr]    = note: removed in 1.92.0; see <https://github.com/rust-lang/rust/pull/138907> for more information
[INFO] [stderr]    = note: merged into `doc_cfg`
[INFO] [stderr] 
[INFO] [stderr] error: Compilation failed, aborting rustdoc
```

## More info

- [Failed build logs](https://docs.rs/crate/oas3/latest/builds/2650481)
- [#[doc(cfg)] tracking issue](https://github.com/rust-lang/rust/issues/43781)
- [PR that replaces `doc_auto_cfg` with `doc_cfg`](https://github.com/rust-lang/rust/pull/138907)

--- a/crates/oas3/src/lib.rs
+++ b/crates/oas3/src/lib.rs
@@ -16,7 +16,7 @@
 //!
 //! [OpenAPI v3.1.x]: https://spec.openapis.org/oas/v3.1.1
 
-#![cfg_attr(docsrs, feature(doc_auto_cfg))]
+#![cfg_attr(docsrs, feature(doc_cfg))]
 
 pub mod spec;
 

